app/src/test/java/com/ably/tracking/demo/publisher/ui/splash/SplashViewModelTest.kt

```diff
@@ -0,0 +1,44 @@
+package com.ably.tracking.demo.publisher.ui.splash
+
+import com.ably.tracking.demo.publisher.BaseViewModelTest
+import com.ably.tracking.demo.publisher.secrets.FakeSecretsManager
+import com.ably.tracking.demo.publisher.ui.FakeNavigator
+import com.ably.tracking.demo.publisher.ui.main.MainActivity
+import com.google.common.truth.Truth
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+internal class SplashViewModelTest : BaseViewModelTest() {
+
+    private val navigator = FakeNavigator()
+
+    private val secretsManager = FakeSecretsManager()
+
+    private val viewModel = SplashViewModel(secretsManager, navigator, baseTestCoroutineDispatcher)
+
+    @Test
+    fun `after calling add on view model new order is created`() = runTest {
```

Comment:
I think this test name is wrong (probably copy-paste issue) :wink:

Reply:
https://github.com/ably/asset-tracking-android-rider-app-demo/pull/39/commits/bdafac00ef69089740b19cdd4d63ca5732750780